packages/sync-actions/src/assets-actions.js

```diff
@@ -0,0 +1,46 @@
+import flatten from 'lodash.flatten'
+
+const REGEX_NUMBER = new RegExp(/^\d+$/)
+const REGEX_UNDERSCORE_NUMBER = new RegExp(/^_\d+$/)
+
+function createRemoveAssetAction(delta) {
+  const assetIdentifier = asset =>
+    asset.id ? { assetId: asset.id } : { assetKey: asset.key }
+  const action = Object.assign(
+    { action: 'removeAsset' },
+    assetIdentifier(delta[0])
+  )
+  return action
+}
+
+function createAddAssetAction(delta) {
+  const asset = delta[0]
+  const action = { action: 'addAsset', asset }
+  return action
+}
+
+export default function actionsMapAssets(diff) {
```

Comment:
Maybe I misread something but is there a reason we don't do it as usually? https://github.com/commercetools/nodejs/blob/master/packages/sync-actions/src/zones-actions.js#L26-L59